uber/templates/registration/register.html

```diff
@@ -16,8 +16,15 @@
             $emailLabel.removeClass('optional-field');
         }
     };
-    $(maybeBold);
-
+    var maybeWarn = function () {
+        var types = {{ c.PRESOLD_ONEDAY_BADGE_TYPES|jsonize }};
+        var badgeType = $.val('badge_type');
+        if (badgeType in types && types[badgeType] !== moment().format('dddd')) {
+            $('#day-warning').text('This badge can ONLY be picked up on ' + types[badgeType] + '.  You cannot pick it up today or any other day than ' + types[badgeType] + '.');
+        } else {
+            $('#day-warning').empty();
+        }
+    };
```

Comment:
We originally talked about putting this check in the ``magprime`` repo, but this seemed generic enough that it could go into the core repo.